inc/azure_macro_utils/macro_utils.h

```diff
@@ -195,6 +195,12 @@ const char* MU_C3(MU_, enumIdentifier,_ToString)(enumIdentifier value)
 
 #define MU_ENUM_TO_STRING_2(enumIdentifier, value) MU_C3(MU_, enumIdentifier,_ToString)(value)
 
+#define MU_PRI_ENUM "s%s (%d)"
```

Comment:
>(%d) [](start = 25, length = 4)

I wish this would be done in MU_ENUM_TO_STRING and MU_ENUM_TO_STRING_2. Because the others (ctest, umock pick that one up). I know - that means allocating something. So maybe we'll have to change all the others (ctest, umock). meh. we'll see.

Reply:
I wish the same. For the same reasons as above, I realized that "mneah, I don't know how to do it". So it is going to stay this way until we have a better approach. Usage of PRI_MU_ENUM is optional anyway.

---
In reply to: [335809915](https://github.com/Azure/azure-macro-utils-c/pull/34#discussion_r335809915) [](ancestors = 335809915)